### DISCUSSION TITLE
Transforming to an organization

### DISCUSSION THREAD

**Discussion by emeryowa** [community/community#23279] [New to GitHub]
<p>Hello</p>
<p>I’m planning to turn our account into an organization. I have read the docs on <a href="https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/converting-a-user-into-an-organization" title="https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/converting-a-user-into-an-organization" rel="nofollow noopener">https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/converting-a-user-into-an-organization</a> but following things are not 100% clear yet</p>
<ol>
<li>The docs says SSH keys of the account will be removed. I assume this includes SSH keys created in user settings &gt; “SSH and GPG keys”? Is this correct? Does this also include any SSH keys created by the user in specific repositories, in Repository &gt; Settings &gt; 'Deploy keys"? </li>
</ol>
<p>What approach do you recommend to keep the keys live? Create new SSH keys under another user?</p>
<ol start="2">
<li>
<p>We are a team of 5 developers. Our hosting provider currently has access to some of our repositories. These are 5 extra people or collaborators. How much will Github charge us per month?</p>
</li>
<li>
<p>Additional question to above: our hosting provider already has an organization. Can we add their organization as a collaborator to some of the repositories? If yes, what impact does this have on our billing?</p>
</li>
<li>
<p>To which account will the repositories be transferred? Will they stay on the account that has been transformed to an organization or be transformed to another account?</p>
</li>
<li>
<p>Does my team need to update remote repository urls after organization transform? Like is recommended in <a href="https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/changing-your-github-username#repository-references" title="https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/changing-your-github-username#repository-references" rel="nofollow noopener">https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/changing-your-github-username#repository-references</a></p>
</li>
</ol>
<p>Thanks in advance</p>

**Comment by wordsoup** [comment, marked as answer]
<p>Hi <a class="mention" href="/emeryowa">@emeryowa</a>,</p>
<p> I’ve provided some answers for you below:</p>
<blockquote>
<p>We are a team of 5 developers. Our hosting provider currently has access to some of our repositories. These are 5 extra people or collaborators. How much will Github charge us per month?</p>
</blockquote>
<p>This will depend on if you’re using Free or our Team plan, which is now a flat $4/user, so we encourage you take a look at our <a href="https://github.com/pricing" title="Pricing" rel="nofollow noopener">pricing</a> to determine which is best for your use case, as we do not provide formal quotes.</p>
<blockquote>
<p>Additional question to above: our hosting provider already has an organization. Can we add their organization as a collaborator to some of the repositories? If yes, what impact does this have on our billing?</p>
</blockquote>
<p>Organizations cannot be added to other organizations. You will need to <a href="https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/adding-outside-collaborators-to-repositories-in-your-organization" title="Adding Outside Collaborators to Your Organization" rel="nofollow noopener">invite</a> individual collaborators to your organization. Your organization’s billing is also not determined by any other account’s billing.</p>
<blockquote>
<p>To which account will the repositories be transferred? Will they stay on the account that has been transformed to an organization or be transformed to another account?</p>
</blockquote>
<p>When <a href="https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/converting-a-user-into-an-organization" title="Converting a User into an Organization" rel="nofollow noopener">converting</a> an individual account into an organization, the repositories do not move. Do keep in mind, while transforming the account to an organization, you will be required to designate a another user account as an organizational owner.</p>
<blockquote>
<p>Does my team need to update remote repository urls after organization transform? Like is recommended in <a href="https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/changing-your-github-username#repository-references" title="https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/changing-your-github-username#repository-references" rel="nofollow noopener">https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/changing-your-github-username#repository-references</a></p>
</blockquote>
<p>Since the repositories will remain in the same location, unless you change the username of your organization, you should not need to update your references. However, if you do plan on changing the organization’s username after converting it, you will need that guide.</p>
<blockquote>
<p>The docs says SSH keys of the account will be removed. I assume this includes SSH keys created in user settings &gt; “SSH and GPG keys”? Is this correct? Does this also include any SSH keys created by the user in specific repositories, in Repository &gt; Settings &gt; 'Deploy keys"? </p>
</blockquote>
<blockquote>
<p>What approach do you recommend to keep the keys live? Create new SSH keys under another user?</p>
</blockquote>
<p>This is correct - all SSH keys will be removed. You will need to create SSH keys under another user’s account.</p>
